traffic_ops/testing/api_contract/v4/request_template.json

```diff
@@ -0,0 +1,24 @@
+{
+	"cdns": [
+		{
+			"name": "test",
+			"domainName": "quest",
+			"dnssecEnabled": false
+		}
+	],
+	"cachegroup": [
+		{
+			"name": "test",
+			"shortName": "test",
+			"latitude": 38.897663,
+			"longitude": -77.036574,
+			"fallbackToClosest": true,
+			"localizationMethods": [
+				"DEEP_CZ",
+				"CZ",
+				"GEO"
+			],
+			"typeId": 23
```

Comment:
You should get this id programmatically using the Type name and making a GET request with the name as a query parameter and then replace the typeId with the returned Id. It wont always be id 23.

Reply:
This is just a template for sample request data, I'm getting this Type programmatically, Please refer to conftest.py lines 443-456.

Reply:
ah okay great that works